<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(149, 103, 21);
background-color: rgb(62, 132, 60);</string>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>71</width>
         <height>461</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(68, 69, 69);</string>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>75</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(68, 69, 69);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image/image-removebg-preview.png</normaloff>../image/image-removebg-preview.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>75</width>
          <height>61</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image/image.png</normaloff>../image/image.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>400</y>
          <width>71</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../image/image-removebg-preview (3).png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>681</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(141, 162, 255);</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>30</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;Adobe Caslon Pro Bold&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>CAFE MANAGEMENT</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>81</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../th__11_-removebg-preview.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 25pt &quot;Adobe Caslon Pro Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Food Corner</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>681</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(141, 162, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>451</width>
          <height>281</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>449</width>
           <height>279</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>0</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>Rating: Sắp xếp tăng dần</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rating: Sắp xếp giảm dần</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>80</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(13, 44, 255);</string>
       </property>
       <property name="text">
        <string>List</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>80</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>80</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Contact</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>80</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">abackground-color: rgb(13, 44, 255);</string>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
